frontend/app/components/layouts/application-layout.tsx

```diff
@@ -0,0 +1,301 @@
+import type { ComponentProps, MouseEvent, ReactNode } from 'react';
+
+import { Link } from '@remix-run/react';
+
+import { faArrowRightFromBracket, faChevronDown, faChevronRight, faCircleUser } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Trans, useTranslation } from 'react-i18next';
+
+import { ButtonLink } from '~/components/buttons';
+import { DropdownMenu, DropdownMenuContent, DropdownMenuItem, DropdownMenuSeparator, DropdownMenuTrigger } from '~/components/dropdown-menu';
+import { InlineLink } from '~/components/inline-link';
+import { LanguageSwitcher } from '~/components/language-switcher';
+import { PageTitle } from '~/components/page-title';
+import { getClientEnv } from '~/utils/env-utils';
+import { scrollAndFocusFromAnchorLink } from '~/utils/link-utils';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { useBreadcrumbs, useBuildInfo, useI18nNamespaces, usePageIdentifier, usePageTitleI18nKey } from '~/utils/route-utils';
+
+export const i18nNamespaces = getTypedI18nNamespaces('gcweb');
+
+export interface ApplicationLayoutProps {
+  children?: ReactNode;
+  showAccountMenu?: boolean;
+}
+
+/**
+ * GCWeb Application page template.
+ * see: https://wet-boew.github.io/GCWeb/templates/application/application-docs-en.html
+ */
+export default function ApplicationLayout({ children, showAccountMenu = false }: ApplicationLayoutProps) {
```

Comment:
React convention with boolean prop that default with `false` is to let them undefined.

Reply:
Fixed